pkg/provider/azure_managedDiskController.go

```diff
@@ -135,13 +137,20 @@ func (c *ManagedDiskController) CreateManagedDisk(options *ManagedDiskOptions) (
 			diskMBpsReadWrite = int64(v)
 		}
 		diskProperties.DiskMBpsReadWrite = to.Int64Ptr(diskMBpsReadWrite)
+
+		if options.LogicalSectorSize != 0 {
+			diskProperties.CreationData.LogicalSectorSize = to.Int32Ptr(options.LogicalSectorSize)
```

Comment:
it's better add a logging here to show `LogicalSectorSize`, that would be easier for debugging, WDYT?

Reply:
Wouldn't it be logged in the azuredisk logs that have the grpc request and response?

Reply:
it will be logged in grpc request, while this is the actual code part taking effect, since there are very few people would use this feature, it's worthwhile add a logging here

Reply:
Added